### ISSUE TITLE
Issues differential expression

### ISSUE THREAD

**Issue by rsalz** [mortazavilab/swan_vis#19]
I noticed that in my diff expression files, there are no negative values for log2fc even when I do not impose filters. I use exactly the same code that you provide in your tutorial except I do not filter e.g. `de_genes = sg.get_de_genes(obs_col, obs_conditions=obs_conditions)`. Not sure what's going wrong..

Also, another maybe more nuanced issue... with long read seq there are a lot of cases where for example the total number of transcripts/reads for a particular gene are very low. Swan still runs diff expression tests on these, resulting in a lot of log2fc that are huge and the same. In the tutorial on the swanvis website you can see this even happens to you, see the +/-297.776029 log2fc numbers in your sample tabes. As these are probably not so interesting, is there a reason they still included in swan output?

**Comment by fairliereese** [comment]
So I tried out the DE module on a SwanGraph I have lying around and I get positive and negative log2fc values, so I am not sure what's going on there either :( If you send me your SwanGraph I can possibly take a look.

With regards to the low read count test, that is definitely a good idea. I will add a read count threshold for these tests to my to-do list for the next release.

As an aside, the DE testing library that Swan uses ([diffxpy](https://github.com/theislab/diffxpy/)), is not really actively supported so I unfortunately recommend using a more robust, better supported tool such as edgeR or DEseq2 to perform differential expression testing in your data. If you want I can point you to some code examples that I have used to run such tests on my long read data.
